### PR TITLE
Fix fetch_artifacts.py script

### PR DESCRIPTION
**Pull request by matislovas**
### Problem
`fetch_artifacts.py` script is broken and needs to be fixed to proceed with release. The error I was seing is:
```
Trying to download build artifacts ...
Error while downloading artifacts from "nightly" pipeline: 'ArtifactsDownloader' object has no attribute 'pipeline_id'
```

### :ballot_box_with_check: Definition of Done checklist
- [x] Commit history is clean ([requirements](../blob/main/docs/git_commit_messages_requirements.md))
- [x] README.md is updated
- [x] Functionality is covered by unit or integration tests

### CODE REVIEW

**Review by olekoliinyk — APPROVED** [review]
+1.5

File: ci/build_libtelio.py [hunk 1]
```diff
@@ -373,6 +374,7 @@ def exec_bindings(args):
         try_download_artifacts(
             args.try_fetch_from_pipeline,
             PROJECT_ROOT,
+            PROJECT_ROOT,
```
Comment (LukasPukenis) [inline ci/build_libtelio.py:377]: Both args are the same, it's confusing a bit

**Review by LukasPukenis — APPROVED** [review]
+0.5 would be nice to have a concrete description of what's broken in the MR description

**Comment by matislovas** [comment]
> +0.5 would be nice to have a concrete description of what's broken in the MR description

Added!

File: ci/build_libtelio.py (hunk 1 above)
Comment (matislovas) [reply ci/build_libtelio.py:377]: Yeah, the idea is the `ArtifactsDownloader` to be re-usable, and those two parameters kind of makes sense.

**Pull request merged**